src/carts/examples.sql

```diff
@@ -0,0 +1,85 @@
+--
```

Comment:
That file is necessary? If is, should be in a proper folder.

Reply:
Fixed.